aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -383,6 +403,19 @@ class QLinearPackWeightFp16Legacy final {
   }
 };
 
+class QLinearPackWeightInt8Onednn final {
+ public:
+  static at::Tensor run(
+    at::Tensor weight, // Not QTensor
+    torch::List<int64_t> input_shape) {
```

Comment:
Is `input_shape` required or not? I think it is optional? Can we clarify this in the comment?